tools-local/prebuilt-baseline-offline.xml

```diff
@@ -260,6 +260,7 @@
     <Usage Id="optimization.windows_nt-x64.PGO.CoreCLR" Version="99.99.99-master-20190505.1" />
     <Usage Id="optimization.windows_nt-x86.IBC.CoreCLR" Version="99.99.99-master-20190505.1" />
     <Usage Id="optimization.windows_nt-x86.PGO.CoreCLR" Version="99.99.99-master-20190505.1" />
+    <Usage Id="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
I want to make sure I understand why this is here.  Is it because there are repos that are hardcoded to Newtonsoft.Json version 9.0.1 and in the past, we were building that version so it wasn't a prebuilt?  What is the plan for eliminating this prebuilt in the future?

Reply:
Arcade utilizes Newtonsoft.json 9.0.1, since it was a part of source build built packages; it needn't be a part of prebuilts before. After sb was updated to build ns.j 12.0.2 and arcade couldn't use it due to sdk stuff it intrinsically still depends on 9.0.1, we needed this 9.0.1 as a prebuilt. As we get closer to preview 7, we hope to have arcade either use systemtext.json or use newer 12.0.2 as sdk updates. 

@crummel , would you like to add anything to the stuff that I probably am vaguely stating?

Reply:
Yes, that's pretty much it.  Don't forget we need to file issues for removing Newtonsoft.Json as a prebuilt for repos that aren't using the source-built version (something like https://github.com/dotnet/cli/issues/11441).